res/css/_common.pcss

```diff
@@ -481,13 +481,43 @@ legend {
     height: unset;
 }
 
-.mx_Dialog_cancelButton {
-    @mixin customisedCancelButton;
```

Comment:
`@mixin` is a legitimate Sass function, and currently there should not be a plan to replace them as far as I know.

Reply:
That mixin is used to make the style of the cancel button on dialogs consistent. Please search the repository for `@mixin customisedCancelButton`, and you will find there are six other lines which use that mixin, such as on `_CompoundDialog.pcss`.

Like @alunturner, it is not really clear to me too what the goal of this PR is.